codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/AWSHttpProtocolServiceClient.kt

```diff
@@ -142,14 +143,37 @@ class AWSHttpProtocolServiceClient(
 
     override fun renderPartitionID() {
         writer.openBlock("public var partitionID: String? {", "}") {
-            writer.write("return \"\\(\$L.clientName) - \\(region ?? \"\")\"", serviceConfig.clientName.toUpperCamelCase())
+            writer.write(
+                "return \"\\(\$L.clientName) - \\(region ?? \"\")\"",
+                serviceConfig.clientName.toUpperCamelCase(),
+            )
         }
         writer.write("")
     }
 
     private val authSchemeResolverDefaultProvider = DefaultProvider(
-        { "Default${AuthSchemeResolverGenerator.getSdkId(ctx)}AuthSchemeResolver()" },
+        { writer.format("Default\$LAuthSchemeResolver()", AuthSchemeResolverGenerator.getSdkId(ctx)) },
         false,
         false
     )
+
```

Comment:
The method below:
- hides `accountId` from being exposed as a config property since it is only to be supplied through credential resolution.
- replaces the string `accountIdEndpointMode` config property with a custom Swift enum that is more user-friendly.